README.md

```diff
@@ -128,21 +133,31 @@ If the previous block was run in a terminal, skip to step 8 and only execute tha
    $ npm run build
    ```
 
-1. Serve the presentation and monitor source files for changes
+1.  Open the .env file, located at the root of the project and ensure that there exists entires for the `JIRA_API_KEY` and `JIRA_URL`. If not copy and paste the following into the file:
+    ```sh
+    JIRA_API_KEY="[PLACE YOUR TOKEN HERE]"
+    JIRA_URL="https://cytoscape.atlassian.net/rest/api/3/issue/"'
+    ```
+    Be sure to replace [PLACE YOUR TOKEN HERE], including the brackets, with your valid API token for Jira.
+
+1. Serve the presentation and monitor source files for changes be doing A or B.
```

Comment:
delete 'do'

Reply:
Which "be" the "Be" in "Be sure" or something else?

Reply:
"Serve the presentation and monitor source files for changes **be** doing A or B." that "be"